packages/lib/src/components/internal/SendCopyToEmail/SendCopyToEmail.tsx

```diff
@@ -18,12 +19,7 @@ export default function SendCopyToEmail(props) {
     return (
         <div className={cx('adyen-checkout__fieldset', 'adyen-checkout__fieldset--sendCopyToEmail', props.classNames)}>
             <Field classNameModifiers={['sendCopyToEmail']} name={'sendCopyToEmail'} useLabelElement={false} addContextualElement={false}>
-                {renderFormField('boolean', {
-                    onChange: toggleEmailField,
-                    label: i18n.get('boleto.sendCopyToEmail'),
-                    name: 'sendCopyToEmail',
-                    value: sendCopyToEmail
-                })}
+                <Checkbox onChange={toggleEmailField} label={i18n.get('boleto.sendCopyToEmail')} name={'sendCopyToEmail'}/>
```

Comment:
Shouldn't the previous `value` property be passed as: `checked: !!value`?
Doesn't it serve as a way to say whether the checkbox comes "pre-checked"?